### ISSUE TITLE
Database repository

### ISSUE THREAD

**Issue by munikes** [skhaz/fastapi-restful#1]
First of all, its a wonderful example of clean architecture with fastapi, pydantic and firestore.
@skhaz Its possible to extend this example with database repository (p.e postgres) and with some adapters.
Thank you very much. Have a good code!

**Comment by skhaz** [comment]
Yes, it is possible. If I had time, I would add, maybe in the future. Thanks for the feedback; I really appreciate it!
